templates/PortfolioSummary.html

```diff
@@ -1,41 +1,66 @@
 {% import 'macros/_account_macros.html' as account_macros %}
 {% import 'macros/_commodity_macros.html' as commodity_macros %}
 
+<style>
+  .tree-table .num.PnL.positive {
+    color: var(--budget-positive);
+  }
+
+  .tree-table .num.PnL.negative {
+    color: var(--budget-negative);
+  }
+
+  .tree-table .num .date {
+    font-style: italic;
+    font-size: x-small;
+  }
+</style>
+
 <h2>Portfolio Summary Report</h2>
 <br />
 {% for title, data in extension.portfolio_accounts() %}
-  {% set types = data[0] %}
```

Comment:
please don't change the indentation.  It makes it harder to review and run 'git annotate' on.  Also, I liked it the way it was